packages/create-block-tutorial-template/plugin-templates/$slug.php.mustache

```diff
@@ -37,7 +37,35 @@
  *
  * @see https://developer.wordpress.org/reference/functions/register_block_type/
  */
+{{^isDynamic}}
```

Comment:
Not a blocker. Should we inject also `isStatic` variable to make those checks easier to follow? Example:
```suggestion
{{#isStatic}}
```